src/main/java/org/apache/cassandra/distributed/shared/InstanceClassLoader.java

```diff
@@ -52,7 +52,9 @@ public class InstanceClassLoader extends URLClassLoader
               || name.startsWith("javax.")
               || name.startsWith("jdk.")
               || name.startsWith("netscape.")
-              || name.startsWith("org.xml.sax.");
+              || name.startsWith("org.xml.sax.")
+              || name.startsWith("org.jboss.byteman")
```

Comment:
These should end with `.` like the other entries to make sure that we don't accidentally match something like `oshi.jnafoo`.  Otherwise, looks good to me.